src/i18n/messages/en-us.js

```diff
@@ -0,0 +1,16 @@
+import { LOCALES } from '../locales';
+
+export default {
+  [LOCALES.ENGLISH]: {
+    'step0.title': 'COVID-19 Self-Assessment',
+    'step0.message1': `Check if you need to be tested for COVID-19. You can also help others use this tool if they are not able.`,
+    'step0.message2': `DOH strongly urges anyone who has symptoms, including a cough, runny nose, fever or sore throat, to self-isolate for 14 days.`,
+    'step0.message3': `To protect yourself while out in public, wash your hands frequently, and maintain a distance of about 2 metres from others.`,
+    'step0.hero-button': 'Launch Self-Assessment',
+    'step0.translate': `This is also available in {language}, more regional languages will be available soon.`,
+    'step0.info': `If you have no symptoms or exposure concerns, but have questions about COVID-19 or novel coronavirus, please visit the following resources`,
+    'step0.faq': `Frequently Asked Questions`,
+    'step0.disclaimer': 'Disclaimer',
+    'step0.disclaimer-message': `Information on this website is adapted from {guidelines} for triaging COVID-19 cases. This web site is not endorsed by, directly affiliated with, maintained, authorized, or sponsored by the Department of Health. We do not provide professional medical advice. If you find any errors or inaccuracies on this website, please submit feedback to our {telegram}.`,
+  }
```

Comment:
this is good. we should be able to externalise this file with the keys, and give these to other people who are willing to help.